app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,48 @@
+class CommentsController < ApplicationController
+  def index
+    @comments = Comment.all(post: @post)
+  end
+
+  def show
+    @post = Post.find(params[:post_id])
+    @comment = Comment.find(params[:id])
+  end
+
+  def new
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.create(comment_params)
+    redirect_to @post
+  end
+
+  def edit
+    @post = Post.find(params[:post_id])
+    @comment = Comment.find(params[:id])
+  end
+
+  def update
+    @post = Post.find(params[:post_id])
+    @comment = Comment.find(params[:id])
+    @comment.update(comment_params)
+    redirect_to @post
+  end
+
+  def destroy
+    @post = Post.find(params[:post_id])
+    @comment = Comment.find(params[:id])
+    @comment.destroy
+    redirect_to @post
+  end
+
+
+
+  private
+  def comment_params
+    params.require(:comment).permit(:commenter,:commenter_nationality, :comment_content)
+  end
```

Comment:
Not that you need to worry about it for this assignment, but if you find that you're defining a lot of attributes with the same prefix -- in this case, `commenter` -- it means you might benefit from a separate `Commenter` or `User` model.